compair/api/classlist.py

```diff
@@ -613,13 +628,15 @@ def post(self, course_uuid):
             .all()
 
         if len(params.get('ids')) != len(user_courses):
-            return {"error": "One or more users are not enrolled in the course"}, 400
+            abort(400, title="Enrollment Update Failed", message="One or more users are not enrolled in the course.")
```

Comment:
edge case in case user manually edits form or multiple users are editing class list (and dropping users, etc)